CHANGELOG.md

```diff
@@ -29,6 +33,9 @@
 * Fixed scrolling and heights for multiple result sets
 * Fixed bug to use the correct database for new query from Object Explorer
 
+### Contributions and "thank you"
+We would like to thank all our users who raised issues.
```

Comment:
I personally don't see the point in adding a section if we're not thanking anyone specifically - feel free to do whatever you think is appropriate though.

Reply:
Just being consistent with past releases